src/ru-RU/guide/mod/patcher.md

```
@@ -35,7 +35,7 @@ data-full-width-responsive="true"> </ins>
 
 ## GE Patcher Basics
 
-Press "F12" when the game starts to open the developer console. In the Console tab, output similar to the following will appear:在控制台选项卡中，会出现类似以下内容的输出：
+Press "F12" when the game starts to open the developer console. In the Console tab, output similar to the following will appear:Press "F12" when the game starts to open the developer console. In the Console tab, output similar to the following will appear:在控制台选项卡中，会出现类似以下内容的输出：
 
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)